base/src/main/java/com/nedap/archie/ArchieLanguageConfiguration.java

```diff
@@ -54,6 +54,10 @@ public static void setThreadLocalLogicalPathLanguage(String language) {
         currentLogicalPathLanguage.set(language);
     }
 
+    public static String getThreadLocalDescriptiongAndMeaningLanguage() {
```

Comment:
Should this be public? I would expect users of this class to use getLogicalPathLanguage or getMeaningAndDescriptionLanguage(). Otherwise it could be possible to keep this public, but then perhaps it needs some javadoc to indicate it should not be used except as a low-level method.